mods/cnc/sequences/structures.yaml

```diff
@@ -435,14 +435,15 @@ wood:
 		Length: 16
 
 gun:
-	idle:
+	idle: gunmake
```

Comment:
What's going on here?

Reply:
We need an invisible placeholder for the sprite body, and the first frame of that animation is empty.

Reply:
Can't we just null out the idle sequence of that WSB?

Reply:
Nope, WSB requires a non-null/empty sequence, and I'm not up to refactoring that (not that I'm sure we should allow WSB to work without a sequence to begin with).

Reply:
Ok. Can we have a comment here instead?

Reply:
Added.